helix-core/src/main/java/org/apache/helix/controller/stages/CurrentStateComputationStage.java

```diff
@@ -248,7 +248,7 @@ private void reportInstanceCapacityMetrics(ClusterStatusMonitor clusterStatusMon
 
         Map<String, ResourceAssignment> currentStateAssignment =
             currentStateOutput.getAssignment(resourceToMonitorMap.keySet());
-        ClusterModel clusterModel = ClusterModelProvider.generateClusterModelFromCurrentState(
+        ClusterModel clusterModel = ClusterModelProvider.generateClusterModelFromExistingAssignment(
```

Comment:
Again, currentState, existingAssignment, bestPossibleState/Assignment... could we please consolidate all the terms/variables to reduce confusion?

Reply:
Please note they are for different usages. Different names are required.